### PR TITLE
set environment variabless for user data dump

### PR DESCRIPTION
**Pull request by GuSilverFlame**
set environment variables needed for the dump generation and restoration to be added to Blockscout on [this issue](https://github.com/poanetwork/blockscout/issues/792)

### CODE REVIEW

File: main/variables.tf [hunk 1]
```diff
@@ -103,6 +103,17 @@ variable "chain_network_icon" {
   }
 }
 
+variable "chain_s3_bucket" {
+  description = "A map of chain names to s3 bucket name"
```
Comment (acravenho) [inline main/variables.tf]: Should we be providing a default value here?

File: modules/stack/config.tf [hunk 2]
```diff
@@ -94,6 +94,20 @@ resource "aws_ssm_parameter" "network_icon" {
   type  = "String"
 }
 
+resource "aws_ssm_parameter" "s3_bucket" {
+  count = "${length(var.chains)}"
+  name  = "/${var.prefix}/${element(keys(var.chains),count.index)}/s3_bucket"
+  value = "${element(values(var.chain_s3_bucket),count.index)}"
```
Comment (acravenho) [inline modules/stack/config.tf]: If you do change the above to a string, don't forget to change this value.

File: modules/stack/variables.tf [hunk 3]
```diff
@@ -38,6 +38,13 @@ variable "chain_network_path" {
 variable "chain_network_icon" {
   default = {}
 }
+variable "chain_s3_bucket" {
+  default = {}
```
Comment (acravenho) [inline modules/stack/variables.tf]: :point_up:

File: main/variables.tf [hunk 4]
```diff
@@ -103,6 +103,17 @@ variable "chain_network_icon" {
...
+}
+
+variable "chain_enable_scheduler" {
+  description = "A map of chain names to \"true\" or empty string to enable/disable scheduler"
+  default = {
+    "sokol" = ""
```
Comment (acravenho) [inline main/variables.tf]: This value cannot be empty. 

```
aws_ssm_parameter.enable_scheduler: error creating SSM parameter: InvalidParameter: 1 validation error(s) found.
- minimum field size of 1, PutParameterInput.Value.
```

**Comment by amandasposito** [comment]
This PR is no longer a priority so we will close it for now.

**Pull request closed without merging**